packages/kbn-legacy-logging/src/log_interceptor.ts

```diff
@@ -130,13 +137,18 @@ export class LogInterceptor extends Stream.Transform {
     return downgradeIfErrorMessage(OPENSSL_GET_RECORD_REGEX, event);
   }
 
+  downgradeIfCertUntrusted(event: AnyEvent) {
+    return downgradeIfErrorMessage(OPENSSL_READ_RECORD_REGEX, event);
```

Comment:
nit: It might be easier to understand the behaviour if we added a `level` paramater to `downgradeIfErrorMessage`. That way `downgradeIfErrorMessage` doesn't need to know about `OPENSSL_READ_RECORD_REGEX` but it's purely generic. And `downgradeIfCertUntrusted` has the "business logic" that `OPENSSL_READ_RECORD_REGEX` should be logged at info where other downgraded errors get downgraded to info

Reply:
> other downgraded errors get downgraded to info
From how I understood the issue,  `downgradeIfCertUntrusted` that should be `info`, whereas `downgradeIfHTTPWhenHTTPS` should log at `debug`.

Reply:
@rudolf moving the business logic to each individual method makes a lot of sense! I've made the changes.

Reply:
> From how I understood the issue, downgradeIfCertUntrusted that should be info, whereas downgradeIfHTTPWhenHTTPS should log at debug.

Yes, sorry that was a type on my part